docs/source/00-introduction/introduction.md

```diff
@@ -1,33 +1,13 @@
-# Introduction to Volto
+# Introduction
 
-Volto is a React-based frontend for content management systems (CMS), currently
-supporting three backend implementations: Plone, Guillotina and a NodeJS
-reference implementation.
+Volto is a React-based frontend for content management systems (CMS), currently supporting three backend implementations: Plone, Guillotina, and a NodeJS reference implementation.
 
-Plone is a CMS built on Python born back in 2001, and unmatched experience in
-the subject.
+Plone is an CMS, which was written in Python back in 2001 and cherishes an unmatched experience in the subject. It has exciting features that are still appealing to developers and users alike. Such as customizable content types, hierarchical URL object traversing, and a sophisticated content workflow powered by a granular permissions model that allows you to build from simple websites to huge complex intranets.
 
-Plone has very interesting features that are still appealing to developers and
-users alike as customizable content types, hierarchical URL object traversing
-and a complex content workflow powered by a granular permissions model that
-allows you to build from simple websites to complex huge intranets.
+Volto exposes all those features and communicates with Plone via its mature REST API. It can be highly themeable and customizable. Volto also supports other APIs like Guillotina, a Python resource management system, which is inspired on Plone using the same basic concepts like traversal, content types, and permissions model.
```

Comment:
Isn't it "inspired by Plone" instead of "inspired on Plone"?

Reply:
@tisto ops! yes, you are right. just fixed it.